var/spack/repos/builtin/packages/rayleigh/package.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Rayleigh(MakefilePackage):
+    __doc__ = """Rayleigh is a 3-D convection code designed for the study of
+    dynamo behavior in spherical geometry."""
```

Comment:
Local style is to avoid the `__doc__` variable.
```suggestion
    """Rayleigh is a 3-D convection code designed for the study of
    dynamo behavior in spherical geometry."""
```